automation_testing/pages/FormFields/FormFieldTextAreaPage.ts

```diff
@@ -0,0 +1,49 @@
+import { Locator, Page } from "@playwright/test";
+import { BasePage } from "../BasePage";
+import { generateRandomId } from "../../utils/helpers/helper";
+
+export class FormFieldTextAreaPage extends BasePage {
+
+	readonly page_path = "formfields-formfieldtextarea--kitchen-sink";
+
+	readonly page: Page;
+	readonly regularTextArea: Locator;
+	readonly maxCharTextArea: Locator;
+	readonly maxCharCounter: Locator;
+	readonly disabledTextArea: Locator;
+	readonly xsSizeTextArea: Locator;
+	readonly smSizeTextArea: Locator;
+	readonly mdSizeTextArea: Locator;
+	readonly lgSizeTextArea: Locator;
+	
+	constructor(page: Page) {
+		super(page);
+		this.page = page;
+		this.regularTextArea = page.locator("textarea#regular");
+		this.maxCharTextArea = page.locator("#withMaxChar");
+		this.maxCharCounter = page.locator("//*[@id='root']/div/form/div[2]/div[2]/div/div/div/div/div[1]/div[1]/div[2]");
+		this.disabledTextArea = page.locator("#disabledExample");
+		this.xsSizeTextArea = page.locator("//*[@id='root']/div/form/div[2]/div[4]/div/div/div/div/div[1]/div[2]/div");
+		this.smSizeTextArea = page.locator("//*[@id='root']/div/form/div[2]/div[5]/div/div/div/div/div[1]/div[2]/div");
+		this.mdSizeTextArea = page.locator("//*[@id='root']/div/form/div[2]/div[6]/div/div/div/div/div[1]/div[2]/div");
+		this.lgSizeTextArea = page.locator("//*[@id='root']/div/form/div[2]/div[7]/div/div/div/div/div[1]/div[2]/div");
+	}
+
+	async visitPage(): Promise<void> {
```

Comment:
Is it possible to move some common methods to the extending BasePage to avoid code duplication? Possible candidates that I see could be common are:
- `visitPage( )`: I think this will be used by all the FormFields
-  `getAutogeneratedText(length: number)`: Used by TextField and TextArea
- `getLimitOfMaxChar( )`: Used by TextField and TextArea
- `getNumberOfCharactersInMaxCharField( )`: Used by TextField and TextArea

If we locate this method in a single place it would be easier if needed to change them only in one place instead of on every FormField page.